src/routes/subscribers.js

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+// import { createquery, getAllqueries, getOneQuery } from "../controlers/queries";
+import { createSubscriber, getAllSubscribers, removeOneSubscriber } from "../controlers/subscribers";
+// import { validateQuery } from "../validations/queries";
+
+const router = express.Router();
+
+// router.get("/", getAllqueries);
+// router.get("/:id", getOneQuery);
```

Comment:
after getting new codes running pls removed the unused codes

Reply:
Thanks. they are now removed